java/sidecar-common/src/main/java/com/instaclustr/sidecar/operations/OperationsResource.java

```diff
@@ -46,6 +46,12 @@ public Operation getOperationById(@NotNull @PathParam("id") final UUID id) {
         return operationsService.operation(id).orElseThrow(NotFoundException::new);
     }
 
+    @POST
```

Comment:
POST requests should modify state. This should be GET request since we're just "getting" a filtered set of operations.

As such, it can probably be merged with the main GET method above.